File: src/gestorAplicacion/usuario/Pasajero.java
package gestorAplicacion.usuario;
import java.util.*;
import java.io.*;

public class Pasajero implements Serializable{
    
    private static final long serialVersionUID = 1L;
    private static ArrayList<Pasajero> pasajeros = new ArrayList<Pasajero>();
    private String nombre;
    private int identificacion;
    private Tiquete tiquete;


    //constructores
    public Pasajero(String nombre, int identificacion) {
        this.nombre = nombre;
        this.identificacion = identificacion;
        this.tiquete = null;
        pasajeros.add(this);
    }

    
    //metodos
    @Override
    public String toString() {
        return nombre+" "+identificacion;
    }


    //getter and setter
    public String getNombre() {
        return nombre;
    }
    public void setNombre(String nombre) {
        this.nombre = nombre;
    }
    public int getIdentificacion() {
        return identificacion;
    }
    public void setIdentificacion(int identificacion) {
        this.identificacion = identificacion;
    }
    public Tiquete getTiquete() {
        return tiquete;
    }
    public void setTiquete(Tiquete tiquete) {
        this.tiquete = tiquete;
    }
    public static ArrayList<Pasajero> getPasajeros() {
        return pasajeros;
    }
    public static void setPasajeros(ArrayList<Pasajero> pasajeros) {
        Pasajero.pasajeros = pasajeros;
    }
    public static long getSerialversionuid() {
        return serialVersionUID;
    }
}

File: Python/src/gestorAplicacion/usuario/Pasajero.py
class Pasajero:
    pasajeros = ArrayList()
    nombre = str()
    identificacion = int()
    tiquete = Tiquete()

    # constructores
    def __init__(self, nombre, identificacion):
        super(Pasajero, self).__init__()
        self.nombre = nombre
        self.identificacion = identificacion
        self.tiquete = None
        self.pasajeros.add(self)

    # metodos
    def __str__(self):
        return self.nombre + " " + self.identificacion

    # getter and setter
    def getNombre(self):
        return self.nombre

    def setNombre(self, nombre):
        self.nombre = nombre

    def getIdentificacion(self):
        return self.identificacion

    def setIdentificacion(self, identificacion):
        self.identificacion = identificacion

    def getTiquete(self):
        return self.tiquete

    def setTiquete(self, tiquete):
        self.tiquete = tiquete

    @classmethod
    def getPasajeros(cls):
        return cls.pasajeros

    @classmethod
    def setPasajeros(cls, pasajeros):
        Pasajero.pasajeros = pasajeros